tpl/service.tpl

```diff
@@ -97,7 +98,7 @@ type {{.RequestTypeName}} struct {
 	{{- if .Deprecated}}
 	// Deprecated since {{.DeprecatedSince.String}}
 	{{- end }}
-	{{.ParamName}} *string {{tick}}json:"{{.Key}},omitempty" url:"{{.Key}},omitempty"{{tick}}
+	{{.ParamName}} string {{- if $post }} {{tick}}json:"{{.Key}}{{ if not .Required}},omitempty{{ end }}"{{tick}} {{- else}} {{tick}}url:"{{.Key}}{{ if not .Required}},omitempty{{ end }}"{{tick}} {{- end}}
```

Comment:
I consider this a breaking change, as the new version would generate a different output for the same input. 
Why do you want to change the field type?

Reply:
the change of the field type has been reverted.